src/uu/install/src/install.rs

```diff
@@ -308,15 +308,25 @@ fn behavior(matches: &ArgMatches) -> Result<Behavior, i32> {
         None
     };
 
+    let backup_mode = backup_control::determine_backup_mode(
+        matches.is_present(OPT_BACKUP_NO_ARG),
+        matches.is_present(OPT_BACKUP),
+        matches.value_of(OPT_BACKUP),
+    );
```

Comment:
What would you think about making `determine_backup_mode` take the whole`matches` instead of three arguments? (I think this version is fine though, too)

Reply:
I already thought about it and that'd be cool, but in that case I'd have to streamline the naming of the various arguments involved across the utilities... I'll try to find the time tomorrow.

Example: 

- install calls the arguments: "backup", "backup2", "suffix"
- cp calls the arguments: "backup", "b", "suffix"

Reply:
You know I gave this some more thought. I'd go even one step further with this: All utils that implement `backup`s functionality have *exactly the same options for that*. So I'd propose that instead of defining the options in each of the applications separately, I may just as well define the backup-related options in the `backup_control` as "finished" clap-Arguments (with all the texts and flags etc.) and then add these to the App in each of the applications. Then I can pass in all `matches` to `determine_backup_mode`, throw errors if the flags weren't present and continue as before if all is well and defined properly.

I think that I'd like to that in a separate PR though, as this is already somewhat big and I'm afraid I may lose myself in the changes... Would that be okay for you or should I attempt to do it in this PR?

Reply:
Agree, doing this in a new PR is fine